modules/serverless-functions-secrets-configmaps-interactively.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Adding the configuration interactively
+
+You can manage secrets and ConfigMaps you access from your function using the `func config` command.
+
+.Procedure
+
+. Run from the function project directory:
++
+[source,terminal]
+----
+$ func config -p <function-project-directory>
+----
++
+Alternatively, specify the function project directory using the `--path` or `-p` option.
+
+. Use the interactive interface to do one of the following operations:
++
+* Environment variables
+** Add
+*** ConfigMap: All key-value pairs as environment variables
+*** ConfigMap: Value from a key
+*** Secret: All key-value pairs as environment variables
+*** Secret: Value from a key
+** Remove
+** List
+* Volumes
+** Add
+*** ConfigMap
+*** Secret
+** Remove
+** List
+
+. Optionally, deploy the function:
```

Comment:
Please check the OCP docs guidelines. I think it goes "Optional. Do x". Is it necessary to include this step at all if it's optional?

Reply:
The last step is good for completeness of the use case, and it is necessary to let the reader know what happens if a secret or config map cannot be accessed.